### DISCUSSION TITLE
Guncelleme

### DISCUSSION THREAD

**Discussion by Fthhaydnn** [IBAX-io/go-ibax#1337] [Soru Cevap - Türkçe]
İbax son guncellemeden önce cüzdan oluşturup twitter hesabımla baglamistim. Guncellemeden sonra key kodları almak için tekrar girdim fakat key kodlarını aldığım wallet adresle ilk açtığım ve Twitter'a bağlandığım wallet adres farklı oldu. Yardımcı olabilir misiniz.

**Comment by Yusufknk** [comment]
Eski cüzdanın kelimelerini ve şifresini yeni cüzdan oluştur kısmından girin. Eski cüzdan gelecektir.

**Comment by Fthhaydnn** [comment]
Tamamdır hallettim teşekkürler.
